openshift/log-anomaly-detector.app.yaml

```diff
@@ -189,6 +191,9 @@ parameters:
   - name: TRAIN_UPDATE_MODEL
     description: "True if updating existing model, False if starting new model"
     value: "True"
+  - name: NUMBER_JOBS
```

Comment:
I'd say also rename this to `PARALLELISM` and use it for `resource` -> `requests` and `limits` for `cpu`. What do you think?